@here/harp-examples/src/styling_extend-a-theme.ts

```diff
@@ -87,13 +87,6 @@ export namespace HelloCustomThemeExample {
                     parkColor: {
                         type: "color",
                         value: "#00aa33"
-                    },
```

Comment:
Can you rewrite this example to showcase the new extend functionality

Reply:
done, I extended the example to show how to override the extruded building style with a rule that renders the buildings with fill.